easybuild/easyblocks/n/neuron.py

```diff
@@ -203,9 +259,12 @@ def make_module_req_guess(self):
         })
 
         if self.with_python:
-            guesses.update({
-                'PYTHONPATH': [self.pylibdir],
-            })
+            if self.cfg['multi_deps']:
+                guesses.update({'EBPYTHONPREFIXES': ['']})
```

Comment:
Why do this in both `make_module_req_guess` and `make_module_extra`, that doesn't make much sense to me?

Reply:
`make_module_req_guess` is where it was initially, but it doesn't produce the expected module lines for some reason. I can remove it from `make_module_req_guess`

Reply:
resolved.